doc/api/http2.md

```diff
@@ -3150,13 +3150,16 @@ will result in a [`TypeError`][] being thrown.
 added: v8.4.0
 -->
 * `headers` {HTTP/2 Headers Object} An object describing the headers
-* `callback` {Function}
-
-Call [`http2stream.pushStream()`][] with the given headers, and wraps the
-given newly created [`Http2Stream`] on `Http2ServerResponse`.
+* `callback` {Function} Called once `http2stream.pushStream()` is finished,
+  or either when the attempt to create the pushed `Http2Stream` has failed or
+  has been rejected, or the state of `Http2ServerRequest` is closed
```

Comment:
The last bit, `, or the state of Http2ServerRequest is closed` should be clarified. `, or the state of Http2ServerRequest is closed prior to calling the pushStream() method.` would be clearer.

Reply:
OK, it's been fixed.